src/net/sourceforge/kolmafia/KoLCharacter.java

```diff
@@ -237,6 +237,7 @@ public enum Gender {
   private static int turnsPlayed = 0;
   private static int currentRun = 0;
   private static long rollover = 0;
+  private static int globalDaycount = 0;
```

Comment:
While I appreciate that this is done everywhere, I don't love it being a static property.

Reply:
I don't either, but other than a large refactor to change the "normal way", doesn't seem like there's a real alternative.

Probably best to be discussed as its own change.

Reply:
I wasn't suggesting changing it everywhere else I was suggesting doing it differently here.